_data/universities/uwo.yml

```diff
@@ -0,0 +1,5 @@
+name: "[Western University](https://news.westernu.ca/2020/03/presidents-statement-on-covid-19-efforts-march-12/)"
```

Comment:
Would you mind renaming the file to `western-university-canada.yml` or something similar? While UWO isn't as likely to encounter collisions as, say, OSU or ASU, I'd like to keep consistent if possible.

Reply:
how about naming universities using their domain names in reverse order, a bit like Apple vendor plists? So, ca.uwo.yml, edu.stanford.yml, etc. It'd avoid collisions and would also provide some visual grouping by university location.

Reply:
Hmm, that seems cool.

What do you think, @toothbrush and @phildini?

Reply:
Tough call!  If it were me, i wouldn't enforce too much, to keep people feeling there's a low barrier to contributing.

But for sure, consistency has the advantage that it's easier for folks to see if they're about to make a duplicate...  Depends on whether you want more mental burden on the maintainers or the contributors!